src/Cache/SchemaCache.php

```diff
@@ -38,14 +45,24 @@ public function getDuration(): int
 
     public function isExclude(string $value): bool
     {
-        return !in_array($value, $this->exclude, true);
+        return in_array($value, $this->exclude, true);
+    }
+
+    public function invalidate(string $cacheTag): void
+    {
+        TagDependency::invalidate($this->cache, $cacheTag);
     }
 
     public function isEnabled(): bool
     {
         return $this->enabled;
     }
 
+    public function set($key, $value, $ttl = null, Dependency $dependency = null): bool
+    {
+        return $this->cache->set($key, $value, $ttl, $dependency);
```

Comment:
It goes against `Psr\SimpleCache\CacheInterface` so `Yiisoft\Cache\CacheInterface` should be used in this class.

Reply:
done.